### ISSUE TITLE
haunted doesn't manage "dashed" name attributes

### ISSUE THREAD

**Issue by felipeegasneris** [matthewp/haunted#55]
☝️ when i want to create a Web Component with a, for example, `<my-wc first-order="342"></my-wc>` the attribute `first-order` cannot be used like an `MyWc.observedAttributes = ['first-order']` i think the "dashed attributes" are the regular way to name webcomponents attributes

**Comment by matthewp** [comment]
Why doesn't dashed attributes work, what problem do you see?

**Comment by felipeegasneris** [comment]
please see my example, https://codepen.io/felipe-egas/pen/vbrxKP?editors=1010

**Comment by jpray** [comment]
The attribute exists, it just isn't converted to camelCase.  Here is a [working codepen](https://codepen.io/johndanielpray/pen/YBvQgO).

```
function MyApp(element) {
  const {normal} = element;
  const myAttr = element['my-attr'];
  return html`
    <h2>my app</h2>
    <p>dashed attribute: ${myAttr}</p>
    <p>normal attribute: ${normal}</p>
  `;
}
```

**Comment by felipeegasneris** [comment]
thanks a lot @jpray ! it's working for me

**Comment by matthewp** [comment]
Let's keep this open, we should probably convert dash cased attributes to camelCased props.

**Comment by matthewp** [comment]
Fixed by #58
